portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -4503,6 +4506,7 @@ definition {
 			rowEntry = "Untitled Form");
 	}
 
+	@ignore = "true"
```

Comment:
@alinedoleron, the SF issue is occurring because `@ignore` should be added following the alphabetic order, e.g. it should be placed after `@description`.